### PR TITLE
docker: simplify apk add lines

### PR DESCRIPTION
**Pull request by keegancsmith**
The images produced are no longer consumed directly by sourcegraph so we don't have security scanners running on them. Additionally we upgraded to the latest alpine now and some of the apk upgrade lines no longer work. Both combined means we can simplify the apk add lines.

Test Plan: built locally

```
docker build -t zoekt .
docker build -t zoekt-indexserver . -f Dockerfile.indexserver
docker build -t zoekt-webserver   . -f Dockerfile.webserver
```

### CODE REVIEW

**Comment by jtibshirani** [comment]
> The images produced are no longer consumed directly by sourcegraph so we don't have security scanners running on them.

Could you explain how this lets us simplify the lines like this? (Apologies if obvious, working on getting up to speed :))

**Comment by keegancsmith** [comment]
> > The images produced are no longer consumed directly by sourcegraph so we don't have security scanners running on them.
> 
> Could you explain how this lets us simplify the lines like this? (Apologies if obvious, working on getting up to speed :))

Our security team came in and added the apk upgrade lines with the specific versions to use since we would get things like trivy complaining about the images. In practice the CVEs didn't actually effect us except for noise in scanners. Now that we don't ship this to customers who complain about these sort of things we can just do simple "apk add" lines on what we need.

**Review by stefanhengl — APPROVED** [review]

**Pull request merged**
